test/lib/sstable_utils.cc

```diff
@@ -66,3 +67,36 @@ sstables::shared_sstable make_sstable_containing(std::function<sstables::shared_
 
     return sst;
 }
+
+std::vector<std::pair<sstring, dht::token>>
+token_generation_for_shard(unsigned tokens_to_generate, unsigned shard,
+                           unsigned ignore_msb, unsigned smp_count) {
```

Comment:
Please don't align broken likes like this. Broken lines should just have two additional indents compared to the line they were broken off from.

Reply:
For the record, I think it looks much better for function declarations as above.

You are probably referring to the "long lines" section in our coding style, and I am fine complying but I think functions and methods should be an exception. That's not the place for this discussion, though

Reply:
Disagree about the looks better, agree about this not being the place for this. We have a truly multi-cultural code-base, plenty of code following this style too.

Reply:
And by trying to enforce yours you are just being racist

Reply:
I'm just stylist.